Image/tests/Image-specs.js

```diff
@@ -0,0 +1,59 @@
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
+import Image from '../Image';
+
+const src = {
+	hd: 'https://via.placeholder.com/200x200',
+	fhd: 'https://via.placeholder.com/300x300',
+	uhd: 'https://via.placeholder.com/600x600'
+};
+
+describe('Image', () => {
```

Comment:
Image.js         |   73.68 |        0 |      75 |   73.68 | 117-124  


Could you please try dispach resize event??
`fireEvent(window, new Event('resize'));`

And please check 
https://stackoverflow.com/questions/65680286/react-testing-library-failing-to-test-window-resize-react-hook
(I didn't try)

Reply:
I already tried the solutions provided on stackOverflow without luck. however, your solution works